modules/nlu/src/views/full/entities/ListEntity.tsx

```diff
@@ -89,9 +91,12 @@ export const ListEntityEditor: React.FC<Props> = props => {
   }
 
   const editOccurrence = (idx: number, occurrence: NLU.EntityDefOccurrence) => {
-    const newSynonym = _.last(occurrence.synonyms)
-    if (!isUnique(newSynonym)) {
-      return toastFailure('Synonyms duplication is not allowed')
+    console.log('Occ', occurrence.synonyms)
+    console.log('State', state.occurrences[idx].synonyms)
+    if (occurrence.synonyms.length > state.occurrences[idx].synonyms.length) {
+      if (!isUnique(_.last(occurrence.synonyms))) {
```

Comment:
please write this as a single condition as there is no need for double indent

occurences.synoyms.length > state.occurences[idx].synonyms.length && !isUnique ......